backend/src/main/java/com/bakdata/conquery/util/SimpleObservable.java

```diff
@@ -0,0 +1,41 @@
+package com.bakdata.conquery.util;
+
+import lombok.NonNull;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class SimpleObservable<T> {
```

Comment:
Die Klasse wird nicht verwendet.

Reply:
@thoniTUB und ich sind darauf einig geworden, dass wir sie später nutzen können

Reply:
Ich würde gerne vermiden ungenutzten Code in dem Projekt mitzuschleifen (wenn dir der Code wichtig ist, mach ein gist draus, dann können wir darauf später zu greifen sollten wir es brauchen). Wenn wir das wieder brauchen werden, wird es sicherlich eine Möglichkeit geben es wieder herzustellen, sind ja nur 40 Codezeilen.

Reply:
@Njimefo Gist/Snippet ist ein guter Platz

Reply:
@thoniTUB @awildturtok  Unter dem folgenden Link kann jeder auf den erzeugten Gist zugreifen 

https://gist.github.com/Njimefo/22be3d8e4a53486276fad172301ca914